easybuild/easyconfigs/h/healthgps/healthgps-1.1.3.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,65 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMakeCp'
+
+name = 'healthgps'
```

Comment:
`name` should be `Health-GPS` ?

Reply:
Indeed, that is what the maintainer suggested as well. I have updated this and the previous version (PR #15245) to the new name.